e2e/disk_setup.go

```diff
@@ -0,0 +1,104 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func diskSetup() func(*testing.T) {
+	return func(t *testing.T) {
+		namespace := TestNamespace
+
+		// get nodes
+		nodeList := &corev1.NodeList{}
+		err := DeployManagerObj.GetCrClient().List(context.TODO(), nodeList, client.HasLabels{labelNodeRoleWorker})
+		if err != nil {
+			t.Fatalf("failed to list nodes: %+v", err)
+		}
+
+		// represents the disk layout to setup on the nodes.
+		var nodeEnv []nodeDisks
+		for _, disks := range nodeList.Items {
+			nodeEnv = []nodeDisks{
+				{
+					disks: []disk{
+						{size: 10},
+						{size: 20},
+					},
+					node: disks,
+				},
+			}
+		}
+
+		/*
+			// represents the disk layout to setup on the nodes.
```

Comment:
Looks like this commented code is not needed. You can remove this.

Reply:
> Added some minor comments. Overall looks good. Need to wait for the CI to pass though. Mostly the review comments are around the disk count. Also, there are some linting errors that should be fixed.

Thanks for the suggestions Santosh. 
Updated the PR.